blueprints/ember-cli-eslint/files/.eslintrc.js

```diff
@@ -2,6 +2,6 @@ var path = require('path');
 
 module.exports = {
   extends: [
-    path.join(__dirname, 'node_modules/ember-cli-eslint/coding-standard/ember-application.js')
+    require.resolve('ember-cli-eslint/coding-standard/ember-application.js')
```

Comment:
in node land its the same with or without, unless there is ambiguity `.json` and `.js` with the same basename